reconcile/prometheus_rules_tester/integration.py

```diff
@@ -122,12 +122,12 @@ def fetch_rule_and_tests(
 def get_rules_and_tests(
     vault_settings: AppInterfaceSettingsV1,
     thread_pool_size: int,
-    cluster_name: Optional[str] = None,
+    cluster_names: Optional[tuple[str]] = None,
```

Comment:
Please use `Iterable` instead of `tuple` here.

Reply:
Hm, but I think click will always create a tuple :thinking:

Reply:
`tuple` is iterable :) Using `Iterable` instead of `tuple` would mean that you can use other iterable types as well, e.g. `list.´`